cmake/sca/polyspace/polyspace-print-console.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python3
+"""
+Print Polyspace results on terminal, for easy review.
+Copyright (C) 2020-2024 The MathWorks, Inc.
+"""
+
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import sys
+from collections import Counter
+
+
+def _parse_findings(filename: str, ignore_metrics=True):
+    """Parse CSV file"""
+    csv_sep = '\t'  # Polyspace default separator
+
+    def consume_header(oneline):
+        parts = oneline.split(csv_sep)
+        header.extend([p.strip() for p in parts])
+
+    def consume_data(oneline):
+        columns = oneline.split(csv_sep)
+        return dict(zip(header, columns, strict=True))
+
+    findings = []
+    header = []
+    with open(filename, encoding="latin-1") as fp:
```

Comment:
Is the result file always in this encoding? Not an issue perse, just odd as utf-8 is considered the default in many areas.

Reply:
We indeed need latin-1 here.